src/Media_Command.php

```diff
@@ -132,21 +132,16 @@ function regenerate( $args, $assoc_args = array() ) {
 			$image_size_filters = $this->add_image_size_filters( $image_size );
 		}
 
-		$errored = false;
-		$successes = $errors = 0;
+		$successes = $errors = $skips = 0;
 		foreach ( $images->posts as $number => $id ) {
-			if ( $this->process_regeneration( $id, $skip_delete, $only_missing, $image_size, ( $number + 1 ) . '/' . $count ) ) {
-				$successes++;
-			} else {
-				$errors++;
-			}
+			$this->process_regeneration( $id, $skip_delete, $only_missing, $image_size, ( $number + 1 ) . '/' . $count, $successes, $errors, $skips );
 		}
 
 		if ( $image_size ) {
 			$this->remove_image_size_filters( $image_size_filters );
 		}
 
-		Utils\report_batch_operation_results( 'image', 'regenerate', count( $images->posts ), $successes, $errors );
+		self::report_batch_operation_results( 'image', 'regenerate', $count, $successes, $errors, $skips );
```

Comment:
Rather than fork this utility, let's instead add a `$skips` parameter to `Utils\report_batch_operation_results()`. We can pass the `$skips` value through, and it will display if WP-CLI is up to date (or degrade gracefully if not).

Reply:
Okay, I'll open a PR on wp-cli for it.

Reply:
https://github.com/wp-cli/wp-cli/pull/4429